app/src/main/java/com/hunguet123/searchImage/data/source/local/room/PhotoLocalDao.kt

```diff
@@ -0,0 +1,16 @@
+package com.hunguet123.searchImage.data.source.local.room
+
+import androidx.room.*
+import com.hunguet123.searchImage.data.model.Photo
+
+@Dao
+interface PhotoLocalDao {
+    @Query("Select * from photoFavorite order by id")
```

Comment:
viết hoa thì viết hoa hẳn, viết thường thì viết thường hẳn 
tốt nhất là viết hoa hết, để phân biệt với tên bảng, tên trường

```suggestion
    @Query("SELECT * FROM photoFavorite ORDER BY id")
```